backend/spotihue/spotihue.py

```diff
@@ -350,21 +350,7 @@ def _convert_xyz_to_xy(self, X: float, Y: float, Z: float) -> Tuple[float, float
 
         return x, y
 
-    def ascertain_track_playing(self) -> bool:
-        """Determines if Spotify is currently playing a track.
-
-        Returns:
-            bool: True if Spotify is playing a track, False otherwise.
-        """
-        current_track = self._get_current_track()
-
-        if current_track is None:
-            return False
-
-        track_is_playing = current_track.get("is_playing")
-        return True if track_is_playing is True else False
-
-    def retrieve_current_track_information(self) -> dict:
+    def retrieve_current_track_information(self, supply_defaults: bool = True) -> dict:
```

Comment:
Can we add a description of the supply_defaults arg to the doc string?

Reply:
Good call! Will do